.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 
 rvm:
-  - 2.3.3
-  - 2.4.0
+  - 2.1.10
```

Comment:
I think you should keep `2.3.3` and `2.4.4`

Reply:
I have kept 2.3 & 2.4, with their latest version (2.3.7).